smoker/client/cli.py

```diff
@@ -35,6 +36,105 @@
     'gray' : '\033[0;37m',
 }
 
+CONFIG_FILE = '/etc/smokercli.yaml'
```

Comment:
~/.smokercli.yaml should be also valid configuration file which should have precedence